wagtail/admin/templates/wagtailadmin/shared/status_tag.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+
+{% if page.url is not None %}
+    <a href="{{ page.url }}" target="_blank" rel="noreferrer" class="status-tag {{classname}}" title="{% trans 'Visit the live page' %}">
+        <span class="visuallyhidden">{% trans "Current page status:" %}</span> {{ page.status_string }}

```

Comment:
Let's pull out this shared bit into a `fragment`, here's an example of this in use

https://github.com/wagtail/wagtail/blob/main/wagtail/admin/templates/wagtailadmin/panels/inline_panel_child.html#L4

Reply:
i think lets not put it in the partial at all ,  putting only ```span``` part of status that has been used through out 
putting it here without any further plan of action might create confusion like ```page_status_tag``` and ```page_status_tag_new```